README.md

```diff
@@ -1,5 +1,5 @@
-Script to install the Brave browser.
+Script to install the Brave browser
 
-It is designed to be piped directly into `sh`.
```

Comment:
Why remove the periods for these last two sentences? They are sentences after all...